src/main/resources/assets/neoforge/lang/en_us.json

```diff
@@ -81,10 +81,13 @@
   "commands.neoforge.setdim.invalid.nochange": "The entity selected (%1$s) is already in the dimension specified (%2$s).",
   "commands.neoforge.setdim.deprecated": "This command is deprecated for removal in 1.17, use %s instead.",
   "commands.neoforge.tps.invalid": "Invalid dimension %1$s Possible values: %2$s",
-  "commands.neoforge.tps.summary.all": "Overall: Mean tick time: %1$s ms. Mean TPS: %2$s",
+  "commands.neoforge.tps.summary.overall": "Overall:",
+  "commands.neoforge.tps.summary.dimension": "'%s':",
+  "commands.neoforge.tps.mean_tps": "Mean TPS: %s",
+  "commands.neoforge.tps.mean_tps.tooltip": "Closer to 20 is better",
+  "commands.neoforge.tps.mean_tick_time": "Mean tick time: %s ms",
+  "commands.neoforge.tps.mean_tick_time.tooltip": "Lower is better, Higher is worse",
   "commands.neoforge.mods.list": "Mod List: %1$s",
-  "commands.neoforge.tps.summary.basic": "Dim %1$s: Mean tick time: %2$s ms. Mean TPS: %3$s",
-  "commands.neoforge.tps.summary.named": "Dim %1$s (%2$s): Mean tick time: %3$s ms. Mean TPS: %4$s",
```

Comment:
Why did you split up the translation keys for the summary lines instead of reworking the existing ones, and applying color to the (`Component`) arguments? 

This is a bit more difficult to visualize when localizing through Crowdin, since one has to know how each part fits together to make the whole, instead of having the whole line be laid out in the translation key.

Reply:
I needed to split up into smaller chunks in order to apply the hover tooltips correctly, its applied to the section not translation arguments.
<sub>Notice how the tooltip in the below gif is visible when hovering more than just the numbers</sub>

Unless you know of a way to apply the hover to correct section while keeping it as 1 translation.
I mean I could apply the hover to the translation arguments but that decreases the hover area significantly making it harder to find (you would only be able to hover over the numbers).

![java_ll2eEw5QXn](https://github.com/user-attachments/assets/bdde0d11-57f0-4bd2-b84d-f5ee0f67c4d3)